client/coral-admin/src/index.js

```diff
@@ -12,13 +14,19 @@ import 'react-mdl/extra/material.js';
 import './graphql';
 import {loadPluginsTranslations, injectPluginsReducers} from 'coral-framework/helpers/plugins';
 
+const eventEmitter = new EventEmitter();
+
+// TODO: pass redux actions through the emitter.
```

Comment:
Yes, I'd love to add that after we are able to use the `store` service from `coral-framework` so I don't have to write duplicate code...